cypress.config.ts

```diff
@@ -9,9 +9,21 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       config.defaultCommandTimeout = 20000
       config.responseTimeout = 20000
+      
 
       // IMPORTANT return the updated config object
       return config
-    }
-  }
```

Comment:
reporter configurations should be outside of the setupNodeEvents function but within the e2e config.